include/mxnet/base.h

```diff
@@ -85,6 +85,18 @@
  */
 #define PROFILER_MESSAGE_FUNCNAME (__FUNCTION__)
 
+/*!
+ * \brief get string from a type_flag (int)
+ */
+#define MSHADOW_TYPE_TO_STRING(type_flag) ::mshadow::TypeFlagStrings[type_flag]
```

Comment:
Thanks for your contribution!
Could you please add a check ‘0 <= type_flag < len(TypeFlagStrings)’?
If the check fails, it will return UnknownType: type_flag.

Reply:
Done